src/AutoPilotPlugins/PX4/PX4AutoPilotPlugin.cc

```diff
@@ -94,6 +95,12 @@ const QVariantList& PX4AutoPilotPlugin::vehicleComponents(void)
                 _tuningComponent->setupTriggerSignals();
                 _components.append(QVariant::fromValue((VehicleComponent*)_tuningComponent));
 
+                if(_vehicle->parameterManager()->parameterExists(_vehicle->id(), "SYS_VEHICLE_RESP")) {
+                    _flightBehavior = new PX4FlightBehavior(_vehicle, this);
+                    _flightBehavior->setupTriggerSignals();
+                    _components.append(QVariant::fromValue((VehicleComponent*)_flightBehavior));
```

Comment:
avoid C cast style, use static_cast, dynamic_cast or reinterpret_cast

Reply:
I was following the file's convention, so only partially related to the PR. I updated all.